jobfunnel/config/valid_options.py

```diff
@@ -1,38 +1,34 @@
 CONFIG_TYPES = {
-    'output_path': [str],
-    'providers': [list],
-    'search_terms': {
-        'region': {
-            'province': [str],
-            'state': [str],
-            'city': [str],
-            'domain': [str],
-            'radius': [int]
+    "output_path": [str],
+    "providers": [list],
+    "search_terms": {
+        "region": {
+            "province": [str],
+            "state": [str],
+            "city": [str],
+            "domain": [str],
+            "radius": [int],
         },
-        'keywords': [list]
+        "keywords": [list],
     },
-    'black_list': [list],
```

Comment:
can we please remove these style changes? They have cluttered the PR diff a fair bit